pyproject.toml

```diff
@@ -1,7 +1,61 @@
 [build-system]
-requires = ["setuptools", "wheel", "cython", "oldest-supported-numpy"]
+requires = [
+    "cython",
+    "oldest-supported-numpy",
+    "setuptools==59.6.0; python_version=='3.6'",
```

Comment:
But don't we need a more recent setuptools version to have this working with the metadata in pyproject.toml? 

(this might be another reason to drop Python 3.6 support)

Reply:
(the wheel build is also still failing for Python 3.6 (the testing step))

Reply:
xref https://github.com/shapely/shapely/issues/1205 about Python 3.6 support